src/app/public/testing/infinite-scroll/infinite-scroll-fixture.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  DebugElement
+} from '@angular/core';
+
+import {
+  ComponentFixture
+} from '@angular/core/testing';
+
+import {
+  SkyAppTestUtility
+} from '@skyux-sdk/testing';
+
+/**
+ * Provides information for and interaction with a SKY UX infinite scroll component.
+ * By using the fixture API, a test insulates itself against updates to the internals
+ * of a component, such as changing its DOM structure.
+ */
+export class SkyInfiniteScrollFixture {
+  private debugElement: DebugElement;
+
+  constructor(
+    private fixture: ComponentFixture<any>,
+    skyTestId: string
+  ) {
+    this.debugElement = SkyAppTestUtility.getDebugElementByTestId(fixture, skyTestId, 'sky-infinite-scroll');
+
+    this.waitForComponentInitialization();
```

Comment:
Will tests work without this? Not all of our components have asynchronous behavior on init, so this may not be needed for all. If it's needed, we should probably have an `await` on it, since it returns a `Promise<void>`.